### DISCUSSION TITLE
Cannot run `Connect to Slynk` command

### DISCUSSION THREAD

**Discussion by nevzatseferoglu** [s-clerc/slyblime#10] [Q&A]
Although having already installed both ```SublimeREPL``` and ```Slyblime``` on Sublime, I cannot run ```Sly: Connect to slynk``` command on the``` command palette``` of Sublime.

**Comment by s-clerc** [comment]
Hi, thanks for trying my plugin !

Sorry, you're unable to use it, I was wondering if you can answer the following questions:

0. Can you see the command but when you run it does nothing or can you not even seen it? 
1. Which platform you're on?
2. Which version of Sublime Text you're using? 
3. If you press ⎈\` (control+\`) you should be able to open the ST console, look there and see if there's anything related to Slyblime. If there is, please send it to me.
4. Did you already do Preferences > Slyblime > Upgrade SublimeREPL to work with Slyblime ?

**Comment by nevzatseferoglu** [comment]
0. I can see the command at the bar section. Nothing is clickable for Slyblime.
1. I'm on Manjaro Linux.
2. Sublime Version: 3.2.2
3. When I built the code, I encountered an error message as ```Unable to find target command: sly_compile_file```. I selected ```Sly``` as a built tool from the ```Tools``` section before.
4. When I go through Preferences / Package Settings / Slyblime , there is an option as ```upgrade SublimeREPL to work with Slyblime``` but it is not clickable.

**Comment by s-clerc** [comment]
> Sublime Version: 3.2.2

Yeah, you need to be using Sublime Text 4 to use the plugin, it's not going to work with 3. I would recommend reïnstalling the plugin if you upgrade to ST4. 

By the way did you install the package from Package Control?

**Comment by nevzatseferoglu** [comment]
Thanks. Yes, I installed both of them.

**Comment by s-clerc** [comment]
Does it work then?

**Comment by nevzatseferoglu** [comment]
No, it does not. Because ```Sublime 3.2.2``` does not allow me to do anything with ```Slyblime``` related contexts.

**Comment by s-clerc** [comment]
Sorry, I meant does it work with Sublime Text 4 now that you installed it? It’s never going to work with 3.2.2 because I used asyncio to develop it, which needs Python 3.5 or superior which is only available with ST4.

**Comment by s-clerc** [comment]
By the way, you can install Sublime Text 4 from these links:

* https://download.sublimetext.com/sublime-text_build-4087_amd64.deb
* https://download.sublimetext.com/sublime_text_build_4087_x64.tar.xz

and then use your key to activate it as it is in beta right now.
